specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/JitPolicies.json

```diff
@@ -0,0 +1,319 @@
+      "get": {
+        "x-ms-examples": {
+          "Get JIT policy": {
+            "$ref": "./examples/JitPolicies/GetJitPolicy_example.json"
+          }
+        },
+        "tags": [
+          "jitPolicies"
+        ],
+        "operationId": "jitPolicies_Get",
+        "description": "Gets a single JIT Policy resource",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceNamespace"
+          },
+          {
+            "$ref": "#/parameters/resourceType"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "$ref": "./common/jitPolicies/jitTypes.json#/parameters/JitNetworkAccessPolicyName"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./common/jitPolicies/jitTypes.json#/definitions/JitNetworkAccessPolicy"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "jitPolicies"
+        ],
+        "x-ms-examples": {
+          "Create JIT policy": {
+            "$ref": "./examples/JitPolicies/CreateJitPolicy_example.json"
+          }
+        },
+        "operationId": "jitPolicies_CreateOrUpdate",
+        "description": "Create a policy for protecting the extended resource using Just-in-Time access control",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceNamespace"
+          },
+          {
+            "$ref": "#/parameters/resourceType"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "$ref": "./common/jitPolicies/jitTypes.json#/parameters/JitNetworkAccessPolicyName"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "./common/jitPolicies/jitTypes.json#/parameters/JitNetworkAccessPolicy"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "$ref": "./common/jitPolicies/jitTypes.json#/definitions/JitNetworkAccessPolicy"
```

Comment:
this is not the schema of the response with 202. you can remove this from here and include it in thee 200 ok response.